MapleServer2/PacketHandlers/Game/InstrumentHandler.cs

```diff
@@ -85,7 +85,7 @@ private static void HandleStartImprovise(GameSession session, PacketReader packe
             InsturmentInfoMetadata instrumentInfo = InstrumentInfoMetadataStorage.GetMetadata(item.Function.Id);
             InstrumentCategoryInfoMetadata instrumentCategory = InstrumentCategoryInfoMetadataStorage.GetMetadata(instrumentInfo.Category);
 
-            Instrument instrument = new Instrument(instrumentCategory.GMId, instrumentCategory.PercussionId, false, session.FieldPlayer.ObjectId)
+            Types.Instrument instrument = new Types.Instrument(instrumentCategory.GMId, instrumentCategory.PercussionId, false, session.FieldPlayer.ObjectId)
```

Comment:
I think so. As I pointed it out on the PR comment, I had to define that I was using Types.Instrument because there was ambiguity. It might ne a .net 6 change.

Reply:
Then try `using Instrument = MapleServer2.Types;` at the top to define the ambiguity.

Reply:
no bueno https://i.imgur.com/QXCuxFe.png

Reply:
I try this PR and i don't have issues with this ambiguity. even not having `using Instrument = MapleServer2.Types;` not sure what's happening, but if .net 6 has a new stuff, you will have to import it as using, but you are not. Either way, at top is `using Instrument = MapleServer2.Types.Instrument;` this have to work.